example_test.go

```diff
@@ -70,6 +70,7 @@ func ExampleConnection_SelectTyped() {
 	// response is [{{} 1111 hello world}]
 }
 
+//nolint
```

Comment:
Here we disable all warnings. However I don't see a reason. I run golangci-lint and don't see warnings from this function.

Reply:
Fixed.